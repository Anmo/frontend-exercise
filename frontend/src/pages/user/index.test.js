import { render } from '@testing-library/react';
import { useRecoilValue } from 'recoil';

import User from './index';

jest.mock('recoil', () => ({
	...jest.requireActual('recoil'),
	useRecoilValue: jest.fn()
}));

test('renders the normal User Page', () => {
	useRecoilValue.mockReturnValue('test');

	const { container } = render(<User />);

	expect(container.firstChild).toMatchSnapshot();
});
