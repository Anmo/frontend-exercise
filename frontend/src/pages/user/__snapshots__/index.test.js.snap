// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the normal User Page 1`] = `user: test`;
